src/examples.jl

```diff
@@ -995,16 +995,40 @@ const _examples = PlotExample[
             scatter!(Point2.(eachcol(rand(d,1000))), alpha=0.25)
         end]
     ),
+    PlotExample(
+	"Mesh3d",
+	"Allows to plot arbitrary 3d meshes. If only x,y,z are given the mesh is generated automatically. You can also specify the connections using the connections keyword. This is only supported on the Plotly backend. The connections are specified using a tuple of arrays where each array contains the index of the first, second and third vertex of a triangle.",
```

Comment:
Please make this a multiline string using triple quotes.

> The connections are specified using a tuple of arrays where each array contains the index of the first, second and third vertex of a triangle

That sounds to me like each array has three elements. Also better to speak about vectors here to not confuse it with matrices. Something like "each vector contains the 0-based indices of one point of a triangle, such that elements at the same position of these vectors form a triangle"

Reply:
Ok, I have updated it